src/ports/postgres/modules/mxgboost/test/madlib_xgboost.sql_in

```diff
@@ -0,0 +1,4246 @@
+(4142,'F',0.65,0.505,0.165,1.357,0.5725,0.281,0.43,11),
+(4143,'M',0.655,0.525,0.18,1.402,0.624,0.2935,0.365,13),
+(4144,'F',0.655,0.5,0.22,1.359,0.642,0.3255,0.405,13),
+(4145,'M',0.67,0.535,0.19,1.669,0.7465,0.2935,0.508,11),
+(4146,'M',0.67,0.525,0.2,1.7405,0.6205,0.297,0.657,11),
+(4147,'M',0.695,0.53,0.21,1.51,0.664,0.4095,0.385,10),
+(4148,'M',0.695,0.55,0.195,1.6645,0.727,0.36,0.445,11),
+(4149,'M',0.77,0.605,0.175,2.0505,0.8005,0.526,0.355,11),
+(4150,'I',0.28,0.215,0.07,0.124,0.063,0.0215,0.03,6),
+(4151,'I',0.33,0.23,0.08,0.14,0.0565,0.0365,0.046,7),
+(4152,'I',0.35,0.25,0.075,0.1695,0.0835,0.0355,0.041,6),
+(4153,'I',0.37,0.28,0.09,0.218,0.0995,0.0545,0.0615,7),
+(4154,'I',0.43,0.315,0.115,0.384,0.1885,0.0715,0.11,8),
+(4155,'I',0.435,0.33,0.095,0.393,0.219,0.075,0.0885,6),
+(4156,'I',0.44,0.35,0.11,0.3805,0.1575,0.0895,0.115,6),
+(4157,'M',0.475,0.37,0.11,0.4895,0.2185,0.107,0.146,8),
+(4158,'M',0.475,0.36,0.14,0.5135,0.241,0.1045,0.155,8),
+(4159,'I',0.48,0.355,0.11,0.4495,0.201,0.089,0.14,8),
+(4160,'F',0.56,0.44,0.135,0.8025,0.35,0.1615,0.259,9),
+(4161,'F',0.585,0.475,0.165,1.053,0.458,0.217,0.3,11),
+(4162,'F',0.585,0.455,0.17,0.9945,0.4255,0.263,0.2845,11),
+(4163,'M',0.385,0.255,0.1,0.3175,0.137,0.068,0.092,8),
+(4164,'I',0.39,0.31,0.085,0.344,0.181,0.0695,0.079,7),
+(4165,'I',0.39,0.29,0.1,0.2845,0.1255,0.0635,0.081,7),
+(4166,'I',0.405,0.3,0.085,0.3035,0.15,0.0505,0.088,7),
+(4167,'I',0.475,0.365,0.115,0.499,0.232,0.0885,0.156,10),
+(4168,'M',0.5,0.38,0.125,0.577,0.269,0.1265,0.1535,9),
+(4169,'F',0.515,0.4,0.125,0.615,0.2865,0.123,0.1765,8),
+(4170,'M',0.52,0.385,0.165,0.791,0.375,0.18,0.1815,10),
+(4171,'M',0.55,0.43,0.13,0.8395,0.3155,0.1955,0.2405,10),
+(4172,'M',0.56,0.43,0.155,0.8675,0.4,0.172,0.229,8),
+(4173,'F',0.565,0.45,0.165,0.887,0.37,0.239,0.249,11),
+(4174,'M',0.59,0.44,0.135,0.966,0.439,0.2145,0.2605,10),
+(4175,'M',0.6,0.475,0.205,1.176,0.5255,0.2875,0.308,9),
+(4176,'F',0.625,0.485,0.15,1.0945,0.531,0.261,0.296,10),
+(4177,'M',0.71,0.555,0.195,1.9485,0.9455,0.3765,0.495,12);
+
+SELECT xgboost(
+    'abalone',  -- Training table
+    'id',       -- Id column
+    'sex',      -- Class label column
+    '*',        -- Independent variables
+    NULL,       -- Columns to exclude from features
+    $$
+    {
+        'learning_rate': [0.01,0.1], #Regularization on weights (eta). For smaller values, increase n_estimators
+        'max_depth': [9,12],#Larger values could lead to overfitting
+        'subsample': [0.85],#introduce randomness in samples picked to prevent overfitting
+        'colsample_bytree': [0.85],#introduce randomness in features picked to prevent overfitting
+        'min_child_weight': [10],#larger values will prevent over-fitting
+        'n_estimators':[100] #More estimators, lesser variance (better fit on test set)
+    }
+    $$,         -- XGBoost grid search parameters
+    'xgb_out',  -- Grid search results table.
+    '',         -- Class weights
```

Comment:
We should also support NULL as an input for Class weights.
Currently it fails with the following error when passing in NULL.
```
ERROR:  spiexceptions.InternalError: TypeError: 'NoneType' object has no attribute '__getitem__' (plpy_elog.c:121)  (seg0 127.0.0.1:6002 pid=78506) (plpy_elog.c:121)
```